targets/TARGET_STM/TARGET_STM32F3/TARGET_STM32F303x8/TARGET_NUCLEO_F303K8/PinNames.h

```diff
@@ -104,6 +104,8 @@ typedef enum {
     LED3        = PB_3,
     LED4        = PB_3,
     USER_BUTTON = 0x20, // no user button on the board
```

Comment:
There is no user button on the small Nucleo 32 pins boards like this one. This definition was probably left by mistake...